pkg/plugins/scorer/kvcache-aware.go

```diff
@@ -8,13 +8,12 @@ import (
 
 	"github.com/redis/go-redis/v9"
 
-	kvcache "github.com/llm-d/llm-d-kv-cache-manager/pkg/kv-cache"
-
 	"sigs.k8s.io/controller-runtime/pkg/log"
-	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/scheduling/plugins"
+	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/scheduling/framework"
 	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/scheduling/types"
-
 	logutil "sigs.k8s.io/gateway-api-inference-extension/pkg/epp/util/logging"
+
+	kvcache "github.com/llm-d/llm-d-kv-cache-manager/pkg/kv-cache"
```

Comment:
nit: shouldn't github.com be before sigs.k8s.io? Neither is the *current* repo...